samples/DispatcherSample/Startup.cs

```diff
@@ -27,47 +26,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
                 await next.Invoke();
             });
 
-            var dictionary = new Dictionary<string, DispatcherFeature>
-            {
-                {
-                    "/example",
-                    new DispatcherFeature
-                        {
-                            Endpoint = new DispatcherEndpoint("example"),
-                            RequestDelegate = async (context) =>
-                            {
-                                await context.Response.WriteAsync("Hello from the example!");
-                            }
-                        }
-                },
-                {
-                    "/example2",
-                    new DispatcherFeature
-                        {
-                            Endpoint = new DispatcherEndpoint("example2"),
-                            RequestDelegate = async (context) =>
-                            {
-                                await context.Response.WriteAsync("Hello from the second example!");
-                            }
-                        }
-                },
-            };
-
-            app.Use(async (context, next) =>
-            {
-                if (dictionary.TryGetValue(context.Request.Path, out var value))
-                {
-                    var dispatcherFeature = new DispatcherFeature
-                    {
-                        Endpoint = value.Endpoint,
-                        RequestDelegate = value.RequestDelegate
-                    };
-
-                    context.Features.Set<IDispatcherFeature>(dispatcherFeature);
-                    await context.Response.WriteAsync("<p>Dispatch</p>");
-                    await next.Invoke();
-                }
-            });
+            app.UseDispatcher();
```

Comment:
Why is this being called when there's a Startup filter?

Reply:
The Startup filter is to call the hidden (terminator) endpoint middleware if this is called. This is where the user is specifying that the Dispatcher should be used. 

/cc @rynowak